src/data/faq.json

```diff
@@ -409,7 +409,9 @@
                                 "anchor": "rec_cert",
                                 "active": false,
                                 "textblock": [
-                                    "Certificates of recovery are supported since version 2.5 of the Corona-Warn-App. Immediate vaccination status display for recovered people: For recovered people, starting with version 2.9, the complete vaccination protection is displayed in the Corona-Warn-App directly after the first vaccination. The two-week waiting period is eliminated for all vaccines except Johnson & Johnson."
+                                    "<b>Updated 30 March, 2022</b>",
+                                    "Certificates of recovery are supported since version 2.5 of the Corona-Warn-App. Immediate vaccination status display for recovered people: For recovered people, starting with version 2.9, the complete vaccination protection is displayed in the Corona-Warn-App directly after the first vaccination. The two-week waiting period is eliminated for all vaccines except Johnson & Johnson.",
+                                    "As of version 2.20, validity dates are no longer displayed on recovery certificates. Since the validity of certificates depends on local rules, the date of the first positive test on which the certificate is based is now displayed instead."

```

Comment:
"recovery certificates" => "certificates of recovery"

for consistency with the first paragraph

Reply:
ok, thanks!